tests/test_bgp_af.rb

```diff
@@ -314,65 +371,59 @@ def test_additional_paths_selection
     #
     # Set and verify
     #
+    if platform == :ios_xr
+      config_ios_xr_dependencies(asn)
+      config('route-policy drop_all', '#drop_all', 'end-policy')
```

Comment:
`'#drop_all'` seems like a no-op to me.  It was pulled in from #87, does anyone know the reason for that command being there?